blaze/utils.py

```diff
@@ -310,3 +310,13 @@ def __get__(self, instance, owner):
             return self
 
         return self._f(instance)
+
+
+class BlazeDeprecationWarning(DeprecationWarning):
```

Comment:
The idea was to so that we can set `warnings.simplefilter` to only display the warning once but I couldn't get it to work.  I created the derived class so that I wouldn't effect how other warnings are displayed, especially since deprecation warnings are ignored by default.